src/components/MoviesList/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import MovieCard from '../MovieCard';
+import { movies } from './model';
+
+const MoviesListComponent = ({ className }) => {
+	return (
+		<div className={className}>
+			{movies.map((movie) => (
+				<MovieCard
+					key={movie.id}
+					title={movie.title}
+					genre={movie.genre}
+					date={movie.date}
+					image={movie.image}
+				/>
+			))}
+		</div>
+	);
+};
+
+MovieCard.propTypes = {
+	title: PropTypes.string.isRequired,
+	genre: PropTypes.string.isRequired,
+	date: PropTypes.number.isRequired,
+	image: PropTypes.string.isRequired,
+};
```

Comment:
You shouldn't keep prop types and default props for MovieCard inside MoviesListComponent, it's better to move it inside the MovieCard component.